persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/tool/Constants.java

```diff
@@ -0,0 +1,6 @@
+package org.cbioportal.persistence.mybatis.tool;
+
+public class Constants {
```

Comment:
Right, I'll change it to PersistenceConstants and will move it to persistence-api.